packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -392,79 +188,180 @@ export default class TraceStatistics extends Component<Props, State> {
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'Avg',
+        dataIndex: 'avg',
+        sorter: (a, b) => sorterFunction({ type: a.type, index: a.avg }, { type: b.type, index: b.avg }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'Min',
+        dataIndex: 'min',
+        sorter: (a, b) => sorterFunction({ type: a.type, index: a.min }, { type: b.type, index: b.min }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'Max',
+        dataIndex: 'max',
+        sorter: (a, b) => sorterFunction({ type: a.type, index: a.max }, { type: b.type, index: b.max }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'ST Total',
+        dataIndex: 'selfTotal',
+        sorter: (a, b) =>
+          sorterFunction({ type: a.type, index: a.selfTotal }, { type: b.type, index: b.selfTotal }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'ST Avg',
+        dataIndex: 'selfAvg',
+        sorter: (a, b) =>
+          sorterFunction({ type: a.type, index: a.selfAvg }, { type: b.type, index: b.selfAvg }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'ST Min',
+        dataIndex: 'selfMin',
+        sorter: (a, b) =>
+          sorterFunction({ type: a.type, index: a.selfMin }, { type: b.type, index: b.selfMin }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'ST Max',
+        dataIndex: 'selfMax',
+        sorter: (a, b) =>
+          sorterFunction({ type: a.type, index: a.selfMax }, { type: b.type, index: b.selfMax }),
+        render: (cell: string) => {
+          return `${cell} ms`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+      {
+        title: 'ST in Duration',
+        dataIndex: 'percent',
+        sorter: (a, b) =>
+          sorterFunction({ type: a.type, index: a.percent }, { type: b.type, index: b.percent }),
+        render: (cell: string) => {
+          return `${cell} %`;
+        },
+        onCell: record => onCellFunction(record),
+      },
+    ];
+
+    const findTablesAccToSelectors = (tableValue: ITableSpan[]): ITableSpan[] => {
+      const isDetailArray: ITableSpan[] = tableValue.filter((val: ITableSpan) => val.isDetail);
```

Comment:
What does isDetail signify?

Reply:
isDetail array means it has all the rows/spans which are belong to the subgroups which we get when we keep valuenameselector2 i.e subgroup attribute in tracestatistics header. So basically if isDetail value of a row is false then it is parent and if isDetail value of a row is true then it is child to the respective parent

Reply:
yes we can call like isChild but the row which is now in subgroup , If we change valuenameselector1 and valuenameselector2 then the same row can now may be in subgroup or it may be a parent.
Like if isDetail means in my view it tells about whether it is detail info i.e in subgroup of a row or it is a row which is grouped .
what would you suggest to use??

Reply:
we're getting bogged down in the details. I am looking at the ITableSpan type declaration, and it has `isDetail` field. What does that field mean? Can we write a concise semantic definition for it (and add it as comment to the type)? For example, this is from HotROD app - which rows here are supposed to have isDetail=true?

<img width="1278" alt="image" src="https://github.com/jaegertracing/jaeger-ui/assets/3523016/cc7b0c8c-d26c-4105-a301-c667c212be71">

Reply:
Does isDetail refer to sub-grouping? E.g. in this pic the 1st row has isDetail=false and 2nd row has isDetail=true? 

<img width="962" alt="image" src="https://github.com/jaegertracing/jaeger-ui/assets/3523016/f7796f08-0eb2-4ff3-ad6e-864212200f99">

Reply:
Ok sure I will add a comment in ITableSpan type declaration about isDetail attribute